src/modules/search/components/form/SearchForm.tsx

```diff
@@ -54,22 +50,16 @@ const SearchForm = ({ onSubmit }: Props) => {
             <div className={styles.divider} />
           </div>
           <div className={styles.row}>
-            <div className={styles.column}>
-              {project_building_type && (
-                <QueryFilter name={FilterName.ProjectBuildingType} {...project_building_type} />
-              )}
-            </div>
-            <div className={styles.column}>
-              {properties && <QueryFilter name={FilterName.Properties} {...properties} />}
-            </div>
-            <div className={styles.column}>
-              {project_new_development_status && (
-                <QueryFilter name={FilterName.ProjectNewDevelopmentStatus} {...project_new_development_status} />
-              )}
-            </div>
+            {(Object.keys(rest) as FilterName[]).map<JSX.Element>((name, index) => (
```

Comment:
I would prefer something else than rest here as it loses the context and it's very hard to know what we are iterating

Reply:
Good point! 👍

Reply:
I changed it to `additionalFilters`